api/py/tox.ini

```diff
@@ -9,10 +9,13 @@ allowlist_externals = rm
 setenv = PYTHONPATH = {toxinidir}:{toxinidir}/test/sample
 # Run a compile test run.
 commands_pre =
-  rm -rf test/sample/production
+  rm -rf test/sample/production/joins/sample_team
+  rm -rf test/sample/production/group_bys/sample_team
```

Comment:
Doesn't the old command do the same?

Reply:
This specifically deletes only sample_team. I introduced another folder for unit_test usage only which is not deleted by tox setup